server/methods/core/orders.js

```diff
@@ -699,12 +699,12 @@ export const methods = {
           });
           // variant image
           if (variantImage) {
-            orderItem.variantImage = path.join(Meteor.absoluteUrl(), variantImage.url());
+            orderItem.variantImage = Meteor.absoluteUrl(variantImage.url());
```

Comment:
I found that Path.join (and [node path module](https://nodejs.org/api/path.html)) is used for file/system paths, and less useful for urls as we have here. (introduced in #1501). The fact that [absoluteUrl](https://docs.meteor.com/api/core.html#Meteor-absoluteUrl) can take a string argument which it joins with the root_url solves this problem better.